pkg/ephemeral/io/carrier.go

```diff
@@ -42,16 +46,47 @@ type Config struct {
 }
 
 // Connect establishes a TCP connection to a socket on a given host and port.
-func (c *Carrier) Connect(ctx context.Context, host, port string) error {
+func (c *Carrier) Connect(playerID int32, ctx context.Context, host string, port string) error {
```

Comment:
Please have a look at this Codacy finding and introduce playerID as second parameter

Reply:
Done